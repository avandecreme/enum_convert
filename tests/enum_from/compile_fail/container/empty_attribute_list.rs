use enum_convert::EnumFrom;

enum Source {
    Unit,
}

#[derive(EnumFrom)]
#[enum_from()] // Should be #[enum_from(Source)]
enum Target {
    #[enum_from]
    Unit,
}

fn main() {}
