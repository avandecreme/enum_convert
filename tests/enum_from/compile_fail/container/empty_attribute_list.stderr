error: expected a list of source enums, for example #[enum_from(Source1, Source2)]
 --> tests/enum_from/compile_fail/container/empty_attribute_list.rs:8:1
  |
8 | #[enum_from()] // Should be #[enum_from(Source)]
  | ^
